### PR TITLE
Dashboard implementation

### PR DESCRIPTION
**Pull request by jbamlee**

### CODE REVIEW

File: api/service/dashboard/dashboard.go [hunk 1]
```diff
@@ -0,0 +1,1155 @@
...
+			SUM(liquidity) AS tvl,
+			t.timestamp
+		FROM (
+			SELECT DISTINCT ON (ps.pair_id, ds.timestamp)
+				ps.pair_id AS pair_id,
+				liquidity0_in_price + liquidity1_in_price AS liquidity,
+				ds.timestamp AS timestamp
+			FROM
+				ds
+			LEFT JOIN pair_stats_30m AS ps ON ps. "timestamp" < ds.timestamp
```
Comment (jhlee-young) [inline api/service/dashboard/dashboard.go:65]: Nit: adding the condition to this join can improve performance
```
 and ps."timestamp" >= ds.timestamp-%d
```
truncBy to %d

about `Execution Time: 1045.355 ms` to `Execution Time: 7.578 ms`

Comment (jhlee-young) [reply api/service/dashboard/dashboard.go:65]: but this condition prevents taking liquidities from before the specified interval..

Comment (jbamlee) [reply api/service/dashboard/dashboard.go:65]: I think we cannot assume pairs always exist between 
`ps."timestamp" >= ds.timestamp-%d` and `ps. "timestamp" < ds.timestamp`

Comment (jbamlee) [reply api/service/dashboard/dashboard.go:65]: > but this condition prevents taking liquidities from before the specified interval.
yes... it was the problem.
I am thinking a better way. Thank you for your suggestion!

File: api/service/dashboard/dashboard.go [hunk 2]
```diff
@@ -0,0 +1,1155 @@
...
+	m := parser.ParsedTx{}
+	query := d.DB.Model(
+		&m,
+	).Select(`
+		COUNT(*) AS tx_count,
+		DATE_TRUNC('day', TO_TIMESTAMP(timestamp)) AS timestamp`,
+	).Where(
+		fmt.Sprintf("%s.chain_id = ?", m.TableName()), d.chainId,
+	).Where(
+		"DATE_TRUNC('day', TO_TIMESTAMP(timestamp)) >= DATE_TRUNC('day', NOW() - INTERVAL '1 month')",
```
Comment (jhlee-young) [inline api/service/dashboard/dashboard.go:1143]: Nit: optimization by reducing calculation
```
timestamp >= extract(epoch from DATE_TRUNC('day', NOW() - INTERVAL '1 month'))
```

Comment (jbamlee) [reply api/service/dashboard/dashboard.go]: https://github.com/dezswap/dezswap-api/pull/17/commits/1c75c68c47760b76a1453b27423553721882d421

**Review by jhlee-young — APPROVED** [review]

**Pull request merged**